HtmlMesh/src/pointer-events-capture-behavior.ts

```diff
@@ -37,73 +32,92 @@ const startCaptureOnEnter = (scene: Scene) => {
     if (captureOnEnterCount === 0) {
         document.addEventListener("pointermove", onPointerMove);
         _scene = _scene ?? scene;
-        let rect = _scene
-            .getEngine()
-            .getRenderingCanvasClientRect() as ClientRect;
-        _canvasRect =
-            _canvasRect ??
-            new DOMRect(rect.left, rect.top, rect.width, rect.height);
+        console.log(
+            "PointerEventsCaptureBehavior: Starting observation of pointer move events."
+        );
+        _scene.onDisposeObservable.add(doStopCaptureOnEnter);
     }
     captureOnEnterCount++;
 };
 
+const doStopCaptureOnEnter = () => {
+    document.removeEventListener("pointermove", onPointerMove);
+    _scene = null;
+    console.log(
+        "PointerEventsCaptureBehavior: Stopping observation of pointer move events."
```

Comment:
Can you use Logger.log from the Babylon logger instead ?

Reply:
Done